server/application-data/application-data.json

```diff
@@ -1,19 +1,67 @@
 {
-    "administrator-credential-list":
-    [
-      {
-        "administrator-id" : "1",
-        "administrator-name" : "administrator",
-        "user-name" : "admin.user",
-        "authorization" : " *** Basic Ath Code *** ",
-        "allowed-methods" : "ALL"
-      },
-      {
-        "administrator-id" : "2",
-        "administrator-name" : "administrator",
-        "user-name" : "admin.user",
-        "authorization" : " *** Basic Ath Code *** ",
-        "allowed-methods" : "PUT"
-      }
-    ]
-  }
\ No newline at end of file
+  "administrator-credential-list": [
```

Comment:
i believe we would need a admin user having "ALL" permission. Could you please add one for such user. We can have it in id "1"

Reply:
done